cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -97,17 +94,25 @@ migrate: ## apply database migrations
 html_coverage: ## generate and view HTML coverage report
 	coverage html && open htmlcov/index.html
 
+# Define PIP_COMPILE_OPTS=-v to get more information during make upgrade.
+PIP_COMPILE = pip-compile --upgrade $(PIP_COMPILE_OPTS)
+
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: piptools ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
```

Comment:
We don't have pip-tools target anymore so we should remove it from upgrade target script

Reply:
Removed the unnecessary target.